deadspace/code/necromorph/necromorphs/species.dm

```diff
@@ -48,6 +48,7 @@
 	inherent_traits = list(
 		TRAIT_CAN_STRIP,
 		TRAIT_NOHUNGER,
+		NOEYESPRITES,
```

Comment:
It should be in `species_traits`

P.S. Thanks for showing that some traits should be moved here from the necro class

Reply:
Fixed. 

Yeah, species_traits and inherent_traits should have all of the necro traits, necro_class traits is kind of a redundancy that is doing the same work twice.